src/main/java/io/vertx/core/VertxOptions.java

```diff
@@ -675,6 +681,28 @@ public VertxOptions setDisableTCCL(boolean disableTCCL) {
     return this;
   }
 
+  /**
+   * Returns whether we want to use daemon vertx thread.
+   *
+   * @return {@code true} means daemon, {@code false} means not daemon(user), {@code null} means do
+   *         not change the daemon option of the created vertx thread.
+   */
+  public Boolean getUseDaemonThread() {
+    return useDaemonThread;
+  }
+
+  /**
+   * Mark the vertx thread as daemon thread or user thread.
+   * <p/>
+   * For keeping the old behavior, the default value is {@code false} instead of {@code null}.
+   *
+   * @param daemon {@code true} means daemon, {@code false} means not daemon(user), {@code null}
+   *        means do not change the daemon option of the created vertx thread.
+   */
+  public void setUseDaemonThread(Boolean daemon) {
```

Comment:
this should be a fluent setter like other setters.

Reply:
Fixed.